# ui_pqExportStateWizard.h is not generated, therefore, it should not have to be installed.
# B. Hajgato May 22 2015
--- ParaView-v4.3.1-source.ORIG/Qt/Components/CMakeLists.txt	2015-01-14 14:07:44.000000000 +0200
+++ ParaView-v4.3.1-source/Qt/Components/CMakeLists.txt	2015-05-22 09:55:27.021704713 +0100
@@ -650,10 +650,10 @@
 #the pqSGExportStateWizard has subclasses that directly access
 #the UI file, and currently we don't have a clean way to break this hard
 #dependency, so for no we install this ui file.
-if(PARAVIEW_INSTALL_DEVELOPMENT_FILES)
-  install(FILES "${CMAKE_CURRENT_BINARY_DIR}/ui_pqExportStateWizard.h"
-          DESTINATION "${VTK_INSTALL_INCLUDE_DIR}")
-endif()
+#if(PARAVIEW_INSTALL_DEVELOPMENT_FILES)
+#  install(FILES "${CMAKE_CURRENT_BINARY_DIR}/ui_pqExportStateWizard.h"
+#          DESTINATION "${VTK_INSTALL_INCLUDE_DIR}")
+#endif()
 
 #IF (NOT PV_INSTALL_NO_DEVELOPMENT)
 #  # Headers
